baselines/pyproject.toml

```
@@ -37,6 +37,10 @@ classifiers = [
     "Typing :: Typed",
 ]
 
+packages = [
+    { include = "flwr_baselines", from = "flwr_baselines" }
+]
+
 [tool.poetry.dependencies]
 python = "^3.8.6"
```

```suggestion
python = "^3.6.2"
```

Let's use the lowest possible.